platform/ios/src/MGLMapView.mm

```diff
@@ -2667,8 +2667,21 @@ - (void)_setVisibleCoordinates:(const CLLocationCoordinate2D *)coordinates count
     {
         latLngs.push_back({coordinates[i].latitude, coordinates[i].longitude});
     }
+    
+    BOOL boundariesCrossDateTime = NO;
+    if (count == 4 && coordinates[1].longitude > 0 && coordinates[3].longitude < 0) {
+        boundariesCrossDateTime = YES;
+        latLngs = [self normalizeCoordinatesForDateTimeChangeSW:coordinates[1] ne:coordinates[3]];
+        
+    }
 
     mbgl::CameraOptions cameraOptions = _mbglMap->cameraForLatLngs(latLngs, padding);
+    
+    if (boundariesCrossDateTime) {
```

Comment:
This code base calls it the “[antimeridian](https://en.wikipedia.org/wiki/Antimeridian)”, not the “date time”.